README.md

```diff
@@ -1,10 +1,10 @@
 # Doctrine Migrations
 
-[![Build Status](https://travis-ci.org/doctrine/migrations.svg)](https://travis-ci.org/doctrine/migrations)
-[![Code Coverage](https://codecov.io/gh/doctrine/migrations/branch/3.1.x/graph/badge.svg)](https://codecov.io/gh/doctrine/migrations/branch/3.1.x)
+[![Build Status](https://github.com/doctrine/migrations/workflows/Continuous%20Integration/badge.svg?label=build)](https://github.com/doctrine/migrations/actions)
+[![Code Coverage](https://codecov.io/gh/doctrine/migrations/branch/3.0.x/graph/badge.svg)](https://codecov.io/gh/doctrine/migrations/branch/3.0.x)
 [![Packagist Downloads](https://img.shields.io/packagist/dm/doctrine/migrations)](https://packagist.org/packages/doctrine/migrations)
 [![Packagist Version](https://img.shields.io/packagist/v/doctrine/migrations)](https://packagist.org/packages/doctrine/migrations)
-[![GitHub license](https://img.shields.io/github/license/doctrine/migrations)](https://github.com/doctrine/migrations/blob/3.1.x/LICENSE)
+[![GitHub license](https://img.shields.io/github/license/doctrine/migrations)](https://github.com/doctrine/migrations/blob/3.0.x/LICENSE)
```

Comment:
you could use a relative link `(LICENSE)` here. Github will resolve it (and Packagist too), and that will avoid having to update it in each branch.